analyzer/libretime_analyzer/message_listener.py

```diff
@@ -142,7 +142,9 @@ def msg_received_callback(self, channel, method_frame, header_frame, body):
         metadata back to the Airtime web application (or report an error).
         """
         logger.info(
-            " - Received '%s' on routing_key '%s'" % (body, method_frame.routing_key)
+            " - Received '{}' on routing_key '{}'".format(
```

Comment:
This could use a f-string

Reply:
I fixed the one.

The tools didn't catch everything, maybe for good reasons or incapability. That said, I don't want to go over the entire code base to fix these. I hope that with time and repetitive editing most of the code will be modernized.